client/src/resources/lang/en_us.json

```diff
@@ -603,220 +603,220 @@
-    "wiki.article.role.deputy.extra":"- You have a basic attack",
-
-    "wiki.article.role.godfather.guide": "Choose a player to kill each night. Choose a backup to kill for you when you are roleblocked. You should try to kill townies who are likely to stop you from winning.",
-    "wiki.article.role.godfather.abilities":"- Visit a player to attack them\n- If you are roleblocked, your backup will attack the first player they visit\n- Choose another living mafia member to be your backup at any time",
-    "wiki.article.role.godfather.attributes":"- You have a basic attack\n- You always have an innocent aura\n- The mafia is told when you are roleblocked if your backup kills for you",
-    "wiki.article.role.godfather.extra":"- If the mafia has no Godfather or Mafioso, then a random mafia member will convert to a godfather\n- If the Godfather dies, their backup will become the Godfather\n- You can not target other mafia members",
-
-    "wiki.article.role.mafioso.guide": "Target players to kill them. You should try to kill townies who are likely to stop you from winning.",
-    "wiki.article.role.mafioso.abilities":"- Visit a player to attack them",
-    "wiki.article.role.mafioso.attributes":"- You have a basic attack",
-    "wiki.article.role.mafioso.extra":"- You can't target other mafia members",
-
-    "wiki.article.role.consort.guide": "Select a player to roleblock and hypnotise them. If you roleblock them, they wont be able to use their ability that night. They will be sent whatever messages you select. You can also choose to not roleblock them, and only send them messages.",
-    "wiki.article.role.consort.abilities":"- Visit a player to roleblock or hypnotise them",
-    "wiki.article.role.consort.attributes":"- You can not be roleblocked\n- You can choose not to roleblock them, and only send them messages, but you will always send at least one message",
-    "wiki.article.role.consort.extra":"- You can't target other mafia members\n- When you roleblock someone, their visits will be cleared",
+    "wiki.article.role.jailor.guide": "During the day, select one player to jail each night. You can talk to your prisoner during the night, and they will not be told who you are. You should interrogate your prisoner to find out what their role is, and if you think they are evil, you can choose to execute them. You may execute up to three times, but if you execute a townie, you can not jail anyone anymore.",
+    "wiki.article.role.jailor.abilities":"* Select one player to jail during the day\n* Select your prisoner to execute them",
+    "wiki.article.role.jailor.attributes":"* You may talk with your prisoner at night\n* If you execute a townie, you forfeit the ability to jail prisoners, and lose all execution uses\n* Players in jail are immune to most abilities while in jail\n* Jailed players can't target or use their abilities\n* Other players are told when the player they visit is jailed",
```

Comment:
You can still use your ability in jail you just cant select people and you are roleblocked

also did you mean target or select

did you mean select or choose? Do you select during night and choose during the day? Does this matter.

People only know if the player they visit is jailed if their ability fails